float32 loc_x
float32 loc_y
float32 orientation
string map

---

float32 loc_x
float32 loc_y
float32 orientation
